nationality-park/park.js

```diff
@@ -0,0 +1,50 @@
+import parksData from "./nationalParksOfPolandData.json" assert { type: "json" };
+
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const name = urlParams.get("parkname");
+
+if (name == null) {
+  window.location.href = "index.html";
+}
+window.addEventListener("DOMContentLoaded", () => {
+  displayData();
+});
+
+const displayData = () => {
+  const parkData = parksData.filter((element) => element.name == name);
+  const description = parkData[0].description;
+  const date = parkData[0].date;
+  const title = parkData[0].title;
+  const symbol = parkData[0].symbol;
+  const area = parkData[0].area;
+  const voivodeship = parkData[0].voivodeship;
+  const splittedText = description.split("Historia");
```

Comment:
bad naming, splittedText is not saying anything about the hold value - splittedDescription would be enough